### PR TITLE
guacamole-default-login

### PR DESCRIPTION
**Pull request by r3dg33k**
### Template / PR Information

- Added guacamole-default-login
- References: https://wiki.debian.org/Guacamole#:~:text=You%20can%20now%20access%20the,password%20are%20both%20%22guacadmin%22.

### Template Validation

I've validated this template locally?
- [x] YES
- [ ] NO

### CODE REVIEW

File: default-logins/guacamole/guacamole-default-login.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+id: guacamole-default-admin
```
Comment (toufik-airane) [inline default-logins/guacamole/guacamole-default-login.yaml:1]: I don't feel that the `-admin` word is explicit enough to describe a default credentials issue.
What do you think about `-login` or `-credentials` instead?

**Review by toufik-airane — APPROVED** [review]
LGTM 👍 

Just a minor request change if you agree to change the id name.

Thank you.

File: default-logins/guacamole/guacamole-default-login.yaml (hunk 1 above)
Comment (r3dg33k) [reply default-logins/guacamole/guacamole-default-login.yaml:1]: both of them are fine with me. i looked default login repo and set -login

**Comment by princechaddha** [comment]
Hello @r3dg33k, Thanks for your contribution.

The matcher here in this template can lead to false positives as on few hosts, the post data can reflect as json data in the response body. So it would be really helpful if you can share the nuclei debug data.

Also, Feel free to join the [discord server](https://discord.com/invite/projectdiscovery) if you have more info that you can share directly over DM.

**Comment by r3dg33k** [comment]
> Hello @r3dg33k, Thanks for your contribution.
> 
> The matcher here in this template can lead to false positives as on few hosts, the post data can reflect as json data in the response body. So it would be really helpful if you can share the nuclei debug data.
> 
> Also, Feel free to join the [discord server](https://discord.com/invite/projectdiscovery) if you have more info that you can share directly over DM.

[INF] Templates loaded: 1 (New: 85)
[INF] [guacamole-default-admin] Dumped HTTP request for http://192.168.216.132:8080

POST /api/tokens HTTP/1.1
Host: 192.168.216.132:8080
User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0
Connection: close
Content-Length: 37
Content-Type: application/x-www-form-urlencoded
Origin: 192.168.216.132:8080
Referer: 192.168.216.132:8080
Accept-Encoding: gzip

username=guacadmin&password=guacadmin
[INF] [guacamole-default-admin] Dumped HTTP response for http://192.168.216.132:8080/api/tokens

HTTP/1.1 200 
Connection: close
Transfer-Encoding: chunked
Content-Type: application/json
Date: Sun, 08 Aug 2021 17:02:35 GMT

{"authToken":"D2B5E96DB964BDACB248208047AFA271060CFC61D6DB370A5DD0FC3B78889FC0","username":"guacadmin","dataSource":"postgresql","availableDataSources":["postgresql","postgresql-shared"]}
[2021-08-08 13:02:34] [guacamole-default-admin] [http] [high] http://192.168.216.132:8080/api/tokens

**Comment by ehsandeep** [comment]
thank you @r3dg33k for adding more info, this is now merged into the project with updated matchers.

**Pull request merged**